Implemented the chat details button

kamalca opened the pull request:
There is not a details button at the upper right-hand corner of chat screens. This button displays a popup showing details such as the day, time, meal period, and dining hall of the match.

kamalca committed:
Implemented the chat details button

The chat details button currently only holds test data.
I made the assumption that the ChatScreenVC would be able to pass the necessary information to the ChatDetailsVC, making a network request unnecessary.

A special thanks to @hirdaygupta for his patience.

HirdayGupta committed:
Merge branch 'development' into chat_details_popup

HirdayGupta committed:
Merge branch 'development' into chat_details_popup

HirdayGupta committed:
Populate pop up fields correctly

HirdayGupta committed:
Add back commented test data

So that @kamalca can easily debug

HirdayGupta committed:
Improve utility func scoping

HirdayGupta reviewed (changes requested):
Only one change requested: make the pop-up smaller if possible. If this seems like it will take up too much time, then let me know and we can make that a TODO for later.

kamalca committed:
Merge branch 'development' into chat_details_popup

kamalca committed:
Merge branch 'chat_details_popup' of https://github.com/ucladevx/UCLAEats-iOS into chat_details_popup

kamalca committed:
Made the popup less tall

Popup was arbitrarily tall before, now it is only big enough to fit all the objects. Real nice and flexible now. Like a yoga instructor.
